src/main/resources/db/migration/V1_78__case_notes_index.sql

```diff
@@ -1,5 +1,2 @@
+create index if not exists case_notes_mapping_dps_id on case_note_mapping (dps_case_note_id);
 alter table case_note_mapping drop constraint case_note_mapping_pkey;
-alter table case_note_mapping drop constraint case_notes_mapping_nomis_id_unique;
-
-alter table case_note_mapping add primary key (nomis_case_note_id);
-create index case_notes_mapping_dps_id on case_note_mapping (dps_case_note_id);
```

Comment:
Rehash this to leave the existing nomis id constraint as it is, 
and removing the PK and creating a new dps non-unique index. 
I think that having a unique nomis id constraint in place of a PK is fine.

Reply:
also the 'if not exists' part means i can create the index independently if too slow